onward/app/views/fragments/rightMostPopularGeoGarnett.scala.html

```diff
@@ -8,39 +8,43 @@
+                                    right-most-popular__byline--opinion
+                                ">@trail.properties.byline</span>
+                }
+                @trail.properties.maybeContent.map { content =>
+                    @if(content.tags.tags.exists(_.id == "tone/news") || content.tags.tags.exists(_.id == "tone/comment")) {
+                        @fragments.contentAgeNotice(ContentOldAgeDescriber(content))
+                    }
+                }
+            </div>
+        </a>
+    </li>
+}
+
 @if(popular.trails.nonEmpty) {
     <div class="js-right-most-popular right-most-popular right-most-popular--image component--rhc hide-on-childrens-books-site" data-component="geo-most-popular" data-importance="-1" data-test-id="right-most-popular">
         <h3 class="content__meta-heading right-most-popular__heading">most viewed @{country.map{ name => s"in $name"}}</h3>
         <ul class="right-most-popular__items u-unstyled" data-link-name="Right hand most popular geo @countryCode">
-            @popular.trails.take(5).zipWithRowInfo.map{ case((trail: PressedContent), row) =>
-                <li class="right-most-popular-item" data-link-name="trail | @row.rowNum">
-                    <a class="right-most-popular-item__url media u-cf" href="@LinkTo{@trail.header.url}">
-                         @trail.trailPicture.map{ imageMedia =>
-                            <div class="right-most-popular-item__img media__img">
-                                <img class="responsiveimg" src="@ImgSrc.getFallbackUrl(imageMedia)" alt="" />
-                            </div>
-                         }
-                        <div class="media__body right-most-popular__content">
-                            <h4 class="right-most-popular-item__headline">
-                                @if(trail.properties.isLiveBlog) {
-                                    <span class="right-most-popular__kicker">Live</span>
-                                }
-                                @trail.header.headline
-                            </h4>
-                            @if(trail.properties.showByline) {
-                                <span class="
-                                    right-most-popular__byline
-                                    right-most-popular__byline--opinion
-                                ">@trail.properties.byline</span>
-                            }
-                            @trail.properties.maybeContent.map { content =>
-                                @if(content.tags.tags.exists(_.id == "tone/news") || content.tags.tags.exists(_.id == "tone/comment")) {
-                                    @fragments.contentAgeNotice(ContentOldAgeDescriber(content))
-                                }
-                            }
-                        </div>
-                    </a>
-                </li>
+            @popular.trails.take(5).zipWithRowInfo.map{ data =>
+                @template1(data._1, data._2)
```

Comment:
Let's rename data to `trailAndRowTuple` or similar and then it's still a bit clearer what we're passing into the template1 without jumping to it.

Reply:
Names updated to something much better.